.github/workflows/unit_tests.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       PYTHON: ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v2
+      - uses: docker-practice/actions-setup-docker@master
```

Comment:
this makes me slightly nervous since one time the `gcloud` action broke since they switched from `master` to `main` - should we maybe pin to a specific version?